lib/actions/invoke.js

```diff
@@ -14,12 +17,20 @@ function processAction(msg, cfg) {
         secretAccessKey: cfg.aws_secret_access_key
     });
     return new Promise((ok, nok) => {
+        let evalResult = msg.body;
         debug('Calling lambda function=%s region=%s', cfg.functionName, cfg.region);
+        const expression = cfg.expression;
+        if (expression) {
+            debug('Evaluating expression="%s"', expression);
+            const compiledExpression = jsonata(expression);
+            evalResult = compiledExpression.evaluate(msg.body);
```

Comment:
Why can't we rely on the mapper to evaluate jsonata expressions?

Reply:
No. And even after latest changes in mapper it won't be able to do that. I assume end of October that may change, but until then it's the only solution